php/get_category_items.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+//This file is autogenerated. See modules.json and autogenerator.py for details
+
+/*
+    get_category_items.php
+
+    MediaWiki API Demos
+    Demo of `Categorymembers` module : List twenty items in a category
+
+    MIT License
+*/
+
+$endPoint = "https://en.wikipedia.org/w/api.php";
+$params = [
+    "action" => "query",
+    "list" => "categorymembers",
+    "cmtitle" => "Category:Physics",
+    "cmlimit" => "20",
+    "format" => "json"
+];
+
+$url = $endPoint . "?" . http_build_query( $params );
+
+$ch = curl_init( $url );
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+$output = curl_exec( $ch );
+curl_close( $ch );
+
+$result = json_decode( $output, true );
+var_dump( $result );
```

Comment:
I see that we are not obtaining further information from data here https://github.com/wikimedia/mediawiki-api-demos/blob/master/python/get_category_items.py, so we can keep this file as it is unless we decide to make changes to the python file as well..